android-sample-kotlin/app/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.logto.demo">
+    package="io.logto.android.sample4k">
```

Comment:
What does `sample4k` stand for, "sample for kotlin"?
How about just "io.log.kotlin.sample"?

Reply:
1. yes, this naming is commonly used in java and android. eg. `log4j`, `slf4j`
2. package name `kotlin` is only used for kotlin. our sample is good to place under the android package, for this sample is a sample of the android project which uses `android-sdk`, not `kotlin-sdk`.